Plugins/Skinning/src/SkinningPlugin.cpp

```diff
@@ -98,6 +158,33 @@ namespace SkinningPlugin
         CORE_ASSERT( m_current, "should be disabled" );
         CORE_ASSERT( newType >= 0 && newType < 3, "Invalid Skinning Type" );
         m_current->setSkinningType( SkinningComponent::SkinningType( newType ) );
+        switch (newType) {
+        case 0:
+        {
+            m_actionLBS->setIcon(QIcon("../../Assets/Images/LB_on.png"));
```

Comment:
Would be safer to use the Qt resource system to avoid explicit path to icons in the applications.
http://doc.qt.io/qt-5/resources.html